### PR TITLE
sord: update to 0.16.16 

### PR DESCRIPTION
**Pull request by androidnisse**
**Summary**
- Allow SORD_API to be defined by the user
- Fix command line help interface of sord_validate
- Fix dependencies in pkg-config file
- Override pkg-config dependency within meson
- Port sord_validate to pcre2
- Switch to external zix dependency

**Test Plan**
- Checked it installed correctly.

**Checklist**

- [X] Package was built and tested against unstable

**Packaging notes**
- Zix added as dependency and added in this pull request.
- Depends on updated serd from this pull request https://github.com/getsolus/packages/pull/2872 so that one needs merging before this one.
- Sratom depends on this so added this to this pull request.

### CODE REVIEW

File: packages/s/sord/package.yml [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
-version    : 0.16.4
-release    : 4
+version    : 0.16.16
+release    : 5
 source     :
-    - git|https://github.com/drobilla/sord.git : v0.16.4
-license    : ISC
+    - https://download.drobilla.net/sord-0.16.16.tar.xz : 257f876d756143da02ee84c9260af93559d6249dd87f317e70ab5fffcc975fd0
+homepage   : https://drobilla.net/software/sord
+license    : 0BSD
```
Comment (EbonJaeger) [inline packages/s/sord/package.yml:7]: It looks like this project uses `0BSD`, `BSD-3-Clause`, and `ISC` [licenses](https://github.com/drobilla/sord/tree/master/LICENSES). I would guess all of them should be listed:

```yaml
license     :
    - 0BSD
    - BSD-3-Clause
    - ISC
```

Comment (EbonJaeger) [reply packages/s/sord/package.yml:7]: Might want to check the other related projects, too.

**Review by EbonJaeger — CHANGES_REQUESTED** [review]

File: packages/s/sord/package.yml (hunk 1 above)
Comment (androidnisse) [reply packages/s/sord/package.yml]: Licenses fixed, the other two did not use the 3-Clause.

**Review by EbonJaeger — APPROVED** [review]
LGTM, thanks!

**Pull request merged**